_posts/2024-12-17-vscode-osd-setup.markdown

```diff
@@ -0,0 +1,282 @@
+---
+layout: post
+title:  "Streamline OpenSearch Dashboards development with VS Code"
+authors:
+ - wronghuy
+ - kolchfa
+date: 2024-12-17
+categories:
+ - technical-post
+meta_keywords: opensearch dashboards development, vs code, jest
+meta_description: Learn how to setup OpenSearch Dashboards with VS Code
```

Comment:
Please update the meta with the following: 

_meta_keywords:_ OpenSearch Dashboards, VS Code, development tools, Jest integration, ESLint, multi-root workspaces, unit testing, OSD codebase

_meta_description:_ Boost OpenSearch Dashboards development with VS Code. Learn to set up efficient workflows for testing, linting, and debugging. Enhance productivity using Jest, ESLint, and multi-root workspaces.